content/data-visualization/dashboards/widgets/cost-explorer-widget.md

```diff
@@ -7,8 +7,24 @@ tags: ["#getting started", "#metrics", "#widgets", "#dashboards"]
 author: Lawrence Lane
 ---
 
+## Visualization
 Options for this widget type include: **period comparison**, **service total**, and **doughnut**.
 
-The Period Comparison option enables you to select a current timeframe (day, month, 6 months, etc) and compare its previous timeframe interval. For example, the below screenshot shows a _current_ 6 month period comparison of `May 16, 2018 – November 15, 201`8 and a _previous_ 6 month interval of `November 16, 2017 – May 15, 2018`.
+![visualization-options](/images/cost-explorer-widget/visualization-options.png)
+
+## Options
+
+- **Top Limit:** Set a limit on the number of services represented in the visualization
+- **Grouped By:** Choose between AWS Service, Account Id, or Availability Zone
+
+## Time Range
+
+Selecting a Time Range defines a scope in time the visualization uses to measure cost.
+
+Using the Period Comparison visualization enables you to select a current timeframe (day, month, 6 months, etc) and compare its previous timeframe interval. For example, the below screenshot shows a _current_ 6 month period comparison of `May 16, 2018 – November 15, 201`8 and a _previous_ 6 month interval of `November 16, 2017 – May 15, 2018`.
```

Comment:
The 8 in 2018 ought to be before the backtick.

Reply:
Fixed!